src/components/BackButton/BackButton.jsx

```diff
@@ -0,0 +1,24 @@
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import calasses from './BackButton.module.css';
+
+const BackButton = ({ backLink, children }) => {
+  return (
+    <Link className={calasses.Link} to={backLink}>
+      <button
+        className={calasses.Button}
+        type='button'
+        /* onClick={()=>{backPath.goBack()}} */
```

Comment:
Don/t forget to remove unused code please

Reply:
I removed comment code.
```suggestion
    <button
      className={calasses.button}
      onClick={goBack}
      type='button'
    >
      {children}
    </button>
```